docs/user/Alerts.md

```diff
@@ -31,7 +52,25 @@ Machine did not reach the “Running” Phase.  Running phase is when the machin
 * Unusual hostname presented by the kubelet on the bootstrap CSR is preventing CSR approval
 
 ### Resolution
-If the machine never became a node, consult the machine troubleshooting guide.
+If the machine never became a node, consult the [machine troubleshooting guide](TroubleShooting.md).
+
+## MachineDeletingTooLong
```

Comment:
If we go with the above suggestion of `MachineNotYetRunning`, perhaps this should be called `MachineNotYetDeleted` 🤔  Naming alerts is hard

Reply:
Works for me. Done.